### DISCUSSION TITLE
ESP-WROM-32 1.8 TFT SPI 128x160 V1.1

### DISCUSSION THREAD

**Discussion by mathidios** [BitMaker-hub/NerdMiner_v2#523] [Q&A]
Hello my friends, help me by telling me how I should connect the wires so that this screen works on this ESP?

![20241117_120020](https://github.com/user-attachments/assets/0ccc6332-05d6-4111-820b-b958741b870d)

**Comment by paelzer** [comment]
You can have a look here, I have done this some time ago:

https://github.com/BitMaker-hub/NerdMiner_v2/discussions/359
